repose-aggregator/components/filters/compression-filter/src/main/scala/org/openrepose/filters/compression/CompressionFilter.scala

```diff
@@ -91,6 +93,7 @@ class CompressionFilter @Inject()(configurationService: ConfigurationService, co
           logger.error("Error with the CompressingFilter", e)
           httpResponse.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR)
       }
+      httpResponse.commitToResponse()
```

Comment:
I can remove this and the header value check in the unit test, but that seems a little wonky to me.

- https://github.com/rackerlabs/repose/blob/969dd2cb2665a8b4d9dfb80dd91a48a02f89ce00/repose-aggregator/components/filters/compression-filter/src/test/scala/org/openrepose/filters/compression/CompressionFilterCompleteTest.scala#L94